src/host/outputStream.cpp

```diff
@@ -375,58 +214,6 @@ void ConhostInternalGetSet::WarningBell()
     _io.GetActiveOutputBuffer().SendNotifyBeep();
 }
 
-// Routine Description:
-// - Performs a "Reverse line feed", essentially, the opposite of '\n'.
-// Return Value:
-// - <none>
-void ConhostInternalGetSet::ReverseLineFeed()
-{
-    auto& screenInfo = _io.GetActiveOutputBuffer();
-    const SMALL_RECT viewport = screenInfo.GetViewport().ToInclusive();
-    const COORD oldCursorPosition = screenInfo.GetTextBuffer().GetCursor().GetPosition();
-    COORD newCursorPosition = { oldCursorPosition.X, oldCursorPosition.Y - 1 };
-    newCursorPosition = screenInfo.GetTextBuffer().ClampPositionWithinLine(newCursorPosition);
-
-    // If the cursor is at the top of the viewport, we don't want to shift the viewport up.
-    // We want it to stay exactly where it is.
-    // In that case, shift the buffer contents down, to emulate inserting a line
-    //      at the top of the buffer.
-    if (oldCursorPosition.Y > viewport.Top)
-    {
-        // Cursor is below the top line of the viewport
-        THROW_IF_NTSTATUS_FAILED(AdjustCursorPosition(screenInfo, newCursorPosition, TRUE, nullptr));

```

Comment:
In this particular situation it wouldn't apply, because `ReverseLineFeed` shouldn't be panning the viewport when it moves the cursor. That said, When it reaches the top, it'll trigger a scroll of the buffer content, and that does do something similar to `InitializeCursorRowAttributes` for the revealed line. See the calls to `_FillRect` and `ResetLineRenditionRange` in the `_ScrollRectVertically` method.